frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -275,25 +276,14 @@ export const experimentLogic = kea<experimentLogicType>({
                 actions.loadExperimentResults()
             }
         },
-        launchExperiment: async () => {
-            const response: Experiment = await api.update(
-                `api/projects/${values.currentTeamId}/experiments/${values.experimentId}`,
-                {
-                    start_date: dayjs(),
-                }
-            )
-            actions.setExperimentId(response.id || 'new')
-            actions.loadExperiment()
+        launchExperiment: () => {
+            actions.updateExperiment({ start_date: dayjs() })
         },
         endExperiment: async () => {
-            const response: Experiment = await api.update(
-                `api/projects/${values.currentTeamId}/experiments/${values.experimentId}`,
-                {
-                    end_date: dayjs(),
-                }
-            )
-            actions.setExperimentId(response.id || 'new')
-            actions.loadExperiment()
+            actions.updateExperiment({ end_date: dayjs() })
+        },
+        archiveExperiment: async () => {
```

Comment:
archiving an experiment right now means I can't see its data, which doesn't seem right 👀

Reply:
fixed itttt, was because I was filtering out all archived experiments in the "get"

Reply:
also changed to regular delete now